### PR TITLE
Auto-generated. Updating Vectara public protos. (8e9e564b1ada0580d2f0…

### PR DESCRIPTION
**Pull request by faiqibrahim**
Adding streaming changes

### CODE REVIEW

File: admin.proto
Comment (tallatshafaat) [inline admin.proto:1]: The changes in this file should not go out yet as they aren't implemented yet. It seems we missed adding them to private stanza. Can you please send a separate PR to move these to private stanza and then rerun the pipeline? Thanks.

**Comment by tallatshafaat** [comment]
@pwoznic You can approve and merge whenever you're ready to publish the docs.

**Review by pwoznic — APPROVED** [review]
LGTM

**Comment by eskibars** [comment]
@pwoznic go ahead and merge if these look good to you
